extensions/binding/org.eclipse.smarthome.binding.digitalstrom/src/main/java/org/eclipse/smarthome/binding/digitalstrom/internal/lib/structure/scene/SceneDiscovery.java

```diff
@@ -52,9 +52,10 @@
     private SceneManager sceneManager;
     private SceneStatusListener discovery = null;
 
-    private final String query = "/json/property/query?query=/apartment/zones/*(ZoneID)/groups/*(group)/scenes/*(scene,name)";
-    private final String reachableScenesQuery = "/json/zone/getReachableScenes?id=";
-    private final String reachableGroupsQuery = "/json/apartment/getReachableGroups?token=";
+    public final static String NAMEND_SCENE_QUERY = "/apartment/zones/*(ZoneID)/groups/*(group)/scenes/*(scene,name)";
+    // TODO: can be add to dSApi
```

Comment:
What about this TODO?

Reply:
The same like the other TODO, no more time to implement it.